diagnosis/README.md

```diff
@@ -4,21 +4,34 @@ This short [guide](https://github.com/Azure/aks-engine/blob/master/docs/howto/tr
 
 Please refer to this [article](https://docs.microsoft.com/en-us/azure/azure-stack/user/azure-stack-solution-template-kubernetes-trouble) for specifics about how the `Kubernetes Cluster` marketplace item works on Azure Stack.
 
+Follow this [article](https://docs.microsoft.com/azure-stack/user/azure-stack-version-profiles-azurecli2) to configure and login to your Azure Stack instance using Azure CLI
+
 ## Gathering logs
 
-The Bash scripts on this directory are aim to simplify the collection of relevant logs from your Kubernetes cluster. Instead of SSH-ing into the cluster nodes, you can simply download and execute script `getkuberneteslogs.sh` and wait for the logs to be saved back into your workstation.  
+The Bash scripts on this directory are aim to simplify the collection of relevant logs from your Kubernetes cluster. Instead of SSH-ing into the cluster nodes, you can simply download and extract the latest [release](https://github.com/msazurestackworkloads/azurestack-gallery/raw/master/diagnosis/Diagnosis.zip) and execute script `getkuberneteslogs.sh`  
```

Comment:
By link to release I meant this

`https://github.com/msazurestackworkloads/azurestack-gallery/releases`